easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,85 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = '2.6.0'
+versionsuffix = '-Python-2.7.12'
+
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/%(version)s/']
+
+# The patch contains two fixes:
+# - The authors forgot to reinstall the correct root configure script (which is 
+#   available as configure.orig).
+# - A correction to Makefile.in.top, install-toolkit target: The last install
+#   created an error message, so it has been replaced by two commands that 
+#   hopefully perform the action that the authors intended.
```

Comment:
@klust We generally include a description of the patch in the patch itself, i.e. at the top of the patch file (any text above the first line that starts with `---` is comments).

Also, please mention the author of the patch, and mention references to external sources that were used to put it together, if any.